records/0002-nodejs-supported-version.md

```diff
@@ -0,0 +1,40 @@
+# Node.js Supported Version
+
+### Current state
+
+As of the date of this PR (March 1st 2024), the only released and maintained version of Express.js is the v4. 
+This version was created and published in 2014 and is supporting (working on) all Node.js version between 0.10 and 21.x - with tests run against all of them.
+
+During the conception phase of Express v5 it was discussed to drop the support of all version below Node.js v4 (released in September 2015) - so supporting all versions from Node.js 4.x to Node.js 21.x, representing 18 versions.
```

Comment:
any particular reason to stick with 4.x, if v5 wasn't released yet, and hence dropping versions wouldn't be extra breaking change?

Reply:
this is the point I want to raise, and basically, the best for me would be
- only support LTS version (18+) but it may be too radical
- so maybe choose to support v14+